tests/acceptance/17_packages/10_new/unsafe/packagesmatching.cf.sub

```diff
@@ -0,0 +1,23 @@
+# Test that updates list cache is not updated too often.
+
+body common control
+{
+    inputs => { "../../../default.cf.sub" };
+    bundlesequence => { default($(this.promise_filename)) };
+
+    package_module => "test_module";
+}
+
+body package_module test_module
+{
+    query_updates_ifelapsed => "5";
+    query_installed_ifelapsed => "5";
+}
+
+bundle agent test
+{
+  packages:
+      "cfe-present-package"
+        policy => "present",
+        version => "latest";
+}
```

Comment:
I *think* you can simply move this into the top level test file, there is no need to do it here separately. It will carry out the step in `test_pass_1`, and when you get to `test_pass_2` it will have happened already.